rct229/data_fns/table_8_4_4_fns.py

```diff
@@ -30,22 +49,20 @@ def table_8_4_4_lookup(phase, kVA):
 
     Parameters
     ----------
-    type : TransformerType
-        Enumerated transformer type
+    phase : str
+        One of the ElectricalPhase enumeration values
     kVA : float
         Transformer capacity in kVA
 
     Returns
     -------
-    float
-        The required transformer efficiency as a decimal value
+    dict
+        { efficiency: float – The required transformer efficiency as a decimal value }
     """
     # Check that the capacity is in range
-    if (
-        kVA < MIN_KVA
-        or (phase == SINGLE_PHASE and kVA > MAX_SINGLE_PHASE_KVA)
-        or (phase == THREE_PHASE and kVA > MAX_THREE_PHASE_KVA)
-    ):
+    try:
```

Comment:
The assert takes the place of the try ... except, so replace lines 63 – 66 with just line 64.